lib/fluent/plugin/out_forward.rb

```diff
@@ -304,7 +304,7 @@ def send_data(node, tag, chunk)
         sock.write tag.to_msgpack  # tag
 
         # beginRaw(size)
-        sz = chunk.size
+        sz = chunk.size(decompressed: :text)
```

Comment:
We can implement this code with compressed chunks with just only one `Chunk#open`, without `Chunk#bytesize(compressed: :text)` like below:

```ruby
sock.write tag.to_msgpack

chunk.open do |chunk_io|
  sz = chunk_io.size
  sock.write [0xdb, sz].pack('CN')
  IO.copy_stream(chunk_io, sock)
end
```

This code repeats the implementation of `Chunk#write_to`, but it's acceptable because `out_forward` is just the core plugin of Fluentd.